src/execution_plan/ops/op_skip.c

```diff
@@ -49,3 +50,8 @@ static OpResult SkipReset(OpBase *ctx) {
 	return OP_OK;
 }
 
+static OpBase *SkipClone(const ExecutionPlan *plan, const OpBase *opBase) {
+	assert(opBase->type == OPType_SKIP);
```

Comment:
this is a static function, only the owner of it (the skip operation has a pointer to it) as such I think this assertion is unnecessary